spec/lib/embed/media_tag_spec.rb

```diff
@@ -30,11 +30,11 @@
     end
 
     it 'includes a data-src attribute for the dash player' do
-      expect(subject).to have_css('[data-src]', count: 2, visible: false)
+      expect(subject).to have_css('[data-src]', count: 3, visible: false)
     end
 
     it 'includes a data attribute that includes the url to check the users auth status' do
-      expect(subject).to have_css('video[data-auth-url]', count: 2, visible: false)
+      expect(subject).to have_css('video[data-auth-url]', count: 3, visible: false)
```

Comment:
Similar question about these dropping, even if an object was no-download, it shouldn't remove the number of video objects we're getting. (although perhaps I'm miss-understanding).